cmd/controller-manager/app/options/replicaset_options.go

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
how about removing the _options suffix from all options go files here? As the package is called options, the suffix is redundant.

Reply:
great! will remove, now that in `option` directory.